config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -170,15 +168,15 @@
 # Run `after_commit` and `after_*_commit` callbacks in the order they are defined in a model.
 # This matches the behaviour of all other callbacks.
 # In previous versions of Rails, they ran in the inverse order.
-# Rails.application.config.active_record.run_after_transaction_callbacks_in_order_defined = true
+Rails.application.config.active_record.run_after_transaction_callbacks_in_order_defined = true
```

Comment:
We have very few cases where there even are more than one of these in a file, and from a quick pass they don't generally interact with each other.